doc/guidelines/e2e.md

```diff
@@ -0,0 +1,102 @@
+## How to write tests
+
+### Locator Guidelines
+
+Before developing E2E tests make sure you have read the [locator guidelines](https://github.com/ElektraInitiative/PermaplanT/doc/guidelines/frontend-locators.md).
+
+### General rules
+
+- Be consistent and minimalistic.
+  - Use the same vocabulary as playwright (click, visible, etc.)
```

Comment:
Only, if we don't have it in the glossary. Being consistent to our glossary is first priority. Can you link to the vocabulary of playwright please?